Packages/Tracking/OpenXR/Runtime/Scripts/OpenXRLeapProvider.cs

```diff
@@ -186,7 +191,7 @@ private bool PopulateLeapHandFromOpenXRJoints(HandTracker handTracker, ref Hand
                     _frameId,
                     (handTracker == HandTracker.Left ? 0 : 1),
                     fingerIndex,
-                    10f, // Fixed for now
+                    timeVisible, // Fixed for now
```

Comment:
The comment is no longer necessary :)

Reply:
Fixed in the amended commit, just an oversight on my behalf!